.gitignore

```diff
@@ -1,9 +1,11 @@
-*.swp
+# Artifacts
+node_modules/
 *.map
-*.bak
 
-# mac items
+# Misc
 .DS_Store
+.*.swp
+*.bak
 
-# node items
-node_modules/
+#node
+package-lock.json
```

Comment:
Seems like your edits remove the last line. What editor are you using? I'm sure you can configure this somewhere.